src/Microsoft.ServiceFabric.Services/Communication/ServiceException.cs

```diff
@@ -20,12 +20,12 @@ public ServiceException()
         }
 
         /// <summary>
-        ///The ActualExceptionType is the type of actual exception thrown
+        /// Gets or Sets the ActualExceptionType is the type of actual exception thrown.
```

Comment:
IT should  just be Gets the ActualExceptionType is the type of actual exception thrown.